package.json

```
@@ -16,6 +16,7 @@
     "prepublishOnly": "npm ci && npm test",
     "prepack": "npm run build",
     "build": "tsc",
+    "build-watch": "tsc -w",
```

nit: I think this would be more aligned with the other scripts.

```suggestion
    "build:dev": "tsc -w",
```